presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergNativeMetadata.java

```diff
@@ -311,7 +312,9 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
                 icebergTable.location(),
                 fileFormat,
                 getCompressionCodec(session),
-                icebergTable.properties());
+                icebergTable.properties(),
+                transaction.replaceSortOrder().apply().fields().stream().map(SortField::fromIceberg)
+                        .collect(toImmutableList()));
```

Comment:
We shouldn't open a transaction action (here is `ReplaceSortOrder`) without committing it, this will prevent any subsequent actions in the same transaction.

Reply:
@hantangwangd Added support to sort orders on IcebergNativeMetadata. and moved the test cases to IcebergDistributedTestBase. But some of the test cases are failing.
<img width="1717" alt="image" src="https://github.com/user-attachments/assets/b4129579-53e9-427c-9801-1e48acd726b0">